packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -389,7 +481,10 @@ export function useContext<T>(
   observedBits: void | number | boolean,
 ): T {
   if (__DEV__) {
-    currentHookNameInDev = 'useContext';
+    currentHookNameInDev = HookDevNames[ContextHook];
+    currentHookType = ContextHook;
+    createWorkInProgressHook();
+    currentHookType = null;
   }
```

Comment:
^ calling out this dev vs prod difference. I think it should be ok?

Reply:
What does this do and why?

Reply:
Right now, we don't add a ContextType Hook to the hooks list on a fiber, we just do a straight  context read. this means you can't trace it's previous/current positions when rendering. this above bit adds it the list (via `createWorkInProgressHook`) so it can track it when it gets called again.